README.md

```diff
@@ -62,18 +61,20 @@ app
 The client side code to support this is very simple too:
 
 ```html
-<script src="node_modules/koa-web-monetization/client.js"></script>
-<script>
-  getMonetizationId('http://localhost:8080/pay/:id')
-    .then(id => {
-      var img = document.createElement('img')
-      var container = document.getElementById('container')
-
-      img.src = '/content/' + id
-      img.width = '600'
-      container.appendChild(img)
-    })
-</script>
+<body>
+  <h1>Hello World</h1>
+  <p>Underneath this is a paid image. It will load if you have web monetization on.</p>
+  <div id="container">
+    <!-- Paid image will be appended here -->
+  <img src="/content" width="600"/>
+  </div>
+  <script src="/client.js"></script>
```

Comment:
can we make this default to `__monetizer/client.js` so it's less likely to collide? and we can have an option to set the path to a custom one as well.

Reply:
If they want to change the receiver path for some reason then it wouldn't be too strange to assume they also want to change the path of client.js

Reply:
Fair - now fixed